core/new-gui/src/app/workspace/types/workflow-websocket.interface.ts

```diff
@@ -82,11 +83,29 @@ export type ResultExportRequest = Readonly<{
   operatorId: string;
 }>;
 
+export type CacheStatusUpdateRequest = LogicalPlan;
```

Comment:
can we explain why those two types are equivalent? maybe add some comments?

Reply:
will change later